schedule/sle-micro/selfinstall.yaml

```diff
@@ -15,10 +15,15 @@ conditional_schedule:
     ENABLE_SELINUX:
       '1':
         - transactional/enable_selinux
+  rt:
+    FLAVOR:
+      'Default-RT-SelfInstall':
```

Comment:
SLE Micro 5.2 has flavor `MicroOS-Image-RT-SelfInstall`

Reply:
ah yeah, leap one you mean.

Reply:
* `openSUSE-Leap-Micro.x86_64-5.2.0-Default-RT-SelfInstall-Build18.1.install.iso` 
* `SUSE-MicroOS.x86_64-5.2.0-Default-RT-SelfInstall-Build8.2.install.iso`

Reply:
yes, good point. I think we kept `MicroOS-Image` since it was inbuild in the bootable image... We can change that, specially before 5.3 starts.

Reply:
so, shall we merge and do that change in OBS sync?